sbt-scalajs-cross/src/main/scala/scalajscross/ScalaJSCrossPlugin.scala

```diff
@@ -0,0 +1,23 @@
+package scalajscross
+
+import org.scalajs.sbtplugin.ScalaJSPlugin
+import org.scalajs.sbtplugin.impl.ScalaJSGroupID
+
+import sbtcross._
+
+import sbt._
+
+import scala.language.implicitConversions
+
+object ScalaJSCrossPlugin extends AutoPlugin {
+  override def requires: Plugins = ScalaJSPlugin
+
```

Comment:
This should also have a trigger, so that it is automatically added to all projects that have `.enablePlugins(ScalaJSPlugin)`, even if they are not `crossProject`s. Otherwise `%%%` will not work correctly in JS-only projects.